src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<SearchResults results={[]} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders results", () => {
+    const results = ["image1", "image2", "image3"];
+    const { asFragment } = render(<SearchResults results={results} />);
```

Comment:
You cant have more than one snapshot test in a single file, if you look at the existing snapshot youll see why. If you want to test the various different outputs rendered when passing different props, just write a test for each as you are doing, but never more than one snapshot per file

For this test you could just query for things like the image alt tags, the image src tags, or that the number of images on the screen and assert based on them